### ISSUE TITLE
Attempting to run Docker for Windows with Windows Docker Containers fails to start

### ISSUE THREAD

**Issue by hicksjacobp** [docker/for-win#5705]
- [x] I have tried with the latest version of my channel (Stable)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: 5E37FBD7-E5B8-4065-81CD-A0AFCAACCC53/20200205214911

### Expected behavior
Docker service is running and can start Windows containers.

### Actual behavior
Docker service fails to start with the message "Time out has expired and the operation has not been completed."

### Information

System information should be available in uploaded diagnostics (see above).
  - Windows Version: **10.0.17763 (1809) x64; managed by corporate IT but I have administrative privileges**
  - Symantec 14 (14.0 RU1 MP1) build 3876 (14.0.3876.1100). I have limited privileges and cannot simply disable this. I don't see anything in the logs that indicates something is being blocked.
  - Docker Desktop Version: **2.2.0.0 (42247)**
    
![image](https://user-images.githubusercontent.com/832739/73888760-d27a8c00-4833-11ea-8eb9-28c226c48c45.png)

  - Are you running inside a virtualized Windows e.g. on a cloud server or on a mac VM: **No**
  - I have not been able to successfully start Docker when trying to use Windows containers. I have been able to successfully start and run Linux containers.
  - I initially installed Docker for Windows and selected to use Linux containers. I then tried to switch to using Windows containers after Docker started and this was unsuccessful. I tried verifying various Windows settings through other help articles (such as removing vmcompute exploit protection noted in #3597 and verifying the following Windows features were enabled: "Containers", "Hyper-V", "Windows Hypervisor Platform"). After not having any success, I re-installed Docker for Windows but selected to use Windows containers. This still has not been successful. Also to note - I've restarted my machine after attempting to change features or settings or installs.

### Steps to reproduce the behavior

  1. Install Docker for Windows, select to use Windows containers
  1. Start Docker for Windows
  1. After a short duration, an error window comes up, specifying the following details:
  ```
Time out has expired and the operation has not been completed.

   at System.ServiceProcess.ServiceController.WaitForStatus(ServiceControllerStatus desiredStatus, TimeSpan timeout)
   at Docker.Backend.Processes.WindowsDockerDaemon.TryToStartService(Settings settings, String args, Dictionary`2 env)
   at Docker.Backend.Processes.WindowsDockerDaemon.Start(Settings settings, String daemonOptions, Boolean useProtectedNamedPipe)
   at Docker.Backend.ContainerEngine.Windows.DoStart(Settings settings, String daemonOptions)
   at Docker.Backend.ContainerEngine.Windows.Start(Settings settings, String daemonOptions, Credential credential)
   at Docker.Backend.BackendNamedPipeServer.<Run>b__8_2(Object[] args)
   at Docker.Core.Pipe.NamedPipeServer.<>c__DisplayClass9_0.<Register>b__0(Object[] parameters)
   at Docker.Core.Pipe.NamedPipeServer.RunAction(String action, Object[] parameters)
```

**Comment by hicksjacobp** [comment]
I found I also had Cb Defense installed. After uninstalling, I was able to start Docker, and build/run a the nanoservercore Windows container. I'm working with my IT to see if setting up rules as described on the [Antivirus software and Docker](https://docs.docker.com/engine/security/antivirus/) page will resolve the issue.

I will note that Docker took significantly less time starting up as compared to Linux containers when Cb Defense was still installed. Since the error message indicates a timeout, is there a configurable timeout that can be tested that relates to starting the docker client and talking to the docker daemon?
